specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/OracleAzureDbPostgreSqlSyncTask.json

```diff
@@ -97,6 +97,120 @@
           "description": "Target server brand version",
           "readOnly": true
         },
+        "validationErrors": {
+          "description": "Validation errors associated with the task",
+          "type": "array",
+          "items": {
+            "$ref": "./Common.json#/definitions/ReportableException"
+          },
+          "readOnly": true
+        },
+        "databaseSchemaMap": {
+          "type": "array",
+          "description": "Mapping of schemas per database",
+          "items": {
+            "type": "object",
+            "properties": {
+              "database": {
+                "type": "string"
+              },
+              "schemas": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "GetUserTablesOracleTaskInput": {
```

Comment:
I think Tasks needs an entry for the new scenarios.

Reply:
Done